src/overlays/actors/ovl_En_Raf/z_en_raf.h

```diff
@@ -2,16 +2,64 @@
 #define Z_EN_RAF_H
 
 #include "global.h"
+#include "objects/object_raf/object_raf.h"
+
+#define EN_RAF_GET_TYPE(thisx) (((thisx)->params >> 12) & 0xF)
+#define EN_RAF_GET_REVIVE_TIMER(thisx) (((thisx)->params >> 7) & 0x1F)
+#define EN_RAF_GET_SWITCH_FLAG(thisx) ((thisx)->params & 0x7F)
+
+typedef enum {
+    /* 0 */ EN_RAF_TYPE_NORMAL,
+    /* 1 */ EN_RAF_TYPE_ALEADY_DEAD,          // Spawns without trap, so it can't eat bombs/player
+    /* 2 */ EN_RAF_TYPE_NO_WATER_INTERACTIONS // Won't produce ripples, and particles won't produce splashes
```

Comment:
I suppose this is never used, since afaik there's only a few of these in the entire game?

Reply:
Yeah, there are only 7 of these things, and every single one is of type `EN_RAF_TYPE_NORMAL`.